lark/exceptions.py

```diff
@@ -14,7 +12,7 @@ class ConfigurationError(LarkError, ValueError):
     pass
 
 
-def assert_config(value, options, msg='Got %r, expected one of %s'):
+def assert_config(value, options: list, msg='Got %r, expected one of %s'):
```

Comment:
This is not a correct typehint. We uses set and `dict.keys()` at multiple point.